Checking the Genuineness of your Skywallet.md

```diff
@@ -0,0 +1,76 @@
+# Checking the Genuineness of your Skywallet
+
+## Checking your Skywallet Shipment Package for Tampering
+
+Once you receive the Skywallet shipment, please check for any physical tampering or damage of any sorts to the package.
```

Comment:
I am not sure whether `shipment` should be scrapped completely here and in the title.